scripts/phar_build.sh

```diff
@@ -5,11 +5,24 @@ set -e
 echo "Installing composer dependencies with --no-dev..."
 composer install --no-dev
 
-echo "Building terminus.phar..."
+VERSION=$(cat .version)
+## if the github action doesn't explicitly set this var
+## make sure this is tagged as a development build
+if [[ -z "${BUILD_RELEASE}" ]]
+then
+    echo "$VERSION-dev" > .version
```

Comment:
Maybe we could strip `-dev` here if `-n "${BUILD_RELEASE}"` instead.

Reply:
From a .bumpversion configuration perspective, it's easier to put"-dev" into the version during the build process than it is to specifically target leaving it out and making sure every version gets bumped correctly with the word DEV in the version. YMMV.